packages/tools-react-native/test/__fixtures__/available-platforms/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@rnx-kit/tools-react-native/available-platform-tests",
+  "version": "0.0.0-dev",
+  "dependencies": {
+    "react-native": "0.0.0-dev",
+    "react-native-codegen": "0.0.0-dev",
+    "react-native-macos": "0.0.0-dev",
+    "react-native-tvos": "0.0.0-dev",
+    "react-native-win32": "0.0.0-dev",
```

```suggestion
    "@office-iss/react-native-win32": "0.0.0-dev",
```

This package is published under the `@office-iss` scope.